plugins/Man/__init__.py

```diff
@@ -0,0 +1,38 @@
+from time import sleep
+import requests
+
+from nextcord.ext import commands
+
+from Plugin import AutomataPlugin
+
+class Man(AutomataPlugin):
+    """Linux man command via man7.org"""
+
+    @commands.command()
+    async def man(self, ctx: commands.Context, search: string = ""):
+        """Searches man7.org for the requested man page"""
+        try:
+            if search == "":
+                await ctx.send("No search request given :(")
+            else:
+                res = []
+                for i in range(1,9):
+                    r = requests.get(url=f"https://man7.org/linux/man-pages/man{i}/{search}.{i}.html")
```

Comment:
can we have some system here to cache search results so we don't have to hit man7.org `n` times per search? can probably use a dictionary stored in the `Man` class and have the keys being the URL, with some method to attempt to see if it is cached, fetch it from the cache, or fetch it fresh and cache it, and return the same data that is required in each circumstance